### ISSUE TITLE
Save states (Memories) don't seem to work at all.

### ISSUE THREAD

**Issue by ifighftdragons** [agg23/openfpga-NES#22]
### Version (or build number)

0.3.0

### Actual Behavior

Save states don't seem to work at all. I've tried it on a bunch of different games already, and none of them work with save states.

Trying it will give the error message: "Save not supported", and will corrupt the game and core, thus forcing a hard reboot of the Pocket.

Tried with common games like Super Mario Bros. 1, DuckTales, Contra and so on. None work.

**Comment by agg23** [comment]
I'm not really sure what's going wrong for you. Please try deleting the `Cores/agg23.NES` folder and copying everything back over.

**Comment by ifighftdragons** [comment]
That worked!

What happened was this:

1. I got an email notification that the core was updated (Yay! :D)
2. Before I managed to update my core, you had already pulled it, or at least was in the process of pulling it.
3. I didn't know this, so I downloaded whatever I could find, which was the Source Code. No Release .zip was available, so I figured I'd just use the files from the dist folder in the source code .zip.
4. All hell broke loose!

Thanks for the help. Case closed.
